package.json

```diff
@@ -30,7 +30,7 @@
     "lodash": "^4.17.15"
   },
   "dependencies": {
-    "dnode": "https://github.com/christianvuerings/dnode/#e08e620b18c9086d47fe68e08328b19465c62fb7",
+    "dnode": "https://github.com/davedoesdev/dnode#fc21193b87d6341466c35fce541e9d7dbf339769",
```

Comment:
@rgoldfinger-quizlet This new commit looks very similar to the one we already have in place. See https://github.com/christianvuerings/dnode/commit/e08e620b18c9086d47fe68e08328b19465c62fb7 which also does the same `weak = require("weak-napi")` change

Reply:
Good point, sorry I missed that.